webapps/frontend/app/views/site/backend-alba-detail.html

```diff
@@ -470,6 +471,12 @@ <h4 class="pull-left" data-i18n="alba:generic.presets"></h4>
                                     <!-- /ko -->
                                 </td>
                                 <td>
+                                    <i data-bind="click: function() { $root.editPreset($data) },
+                                                  style: { color: (!inUse) ? 'black' : 'lightgrey' },
```

Comment:
The style still depends on the inUse. I think the style binding can be removed completely, as the "button" won't get disabled anyway.